### DISCUSSION TITLE
Can it interact with other plugins?

### DISCUSSION THREAD

**Discussion by sokollondon** [facelessuser/RegReplace#104] [Q&A] [answered]
After replacement, I need to use the plugin [Incrementor](https://github.com/born2c0de/Incrementor).
To replace e.g. 
```
1.
1.
1.
```
with 
```
1.
2.
3.
```

**Comment by facelessuser** [comment, marked as answer]
Sounds like you want something like the  [Chain of Command plugin](https://packagecontrol.io/packages/Chain%20of%20Command).

RegReplace doesn't directly allow configuring other external commands, so you either need to use something that can chain commands, or write a plugin yourself that specifically chains the commands you are interested in.
